### PR TITLE
adding code coverage back

### PR DESCRIPTION
**Pull request by laerus**
@gforsyth ping!

### CODE REVIEW

**Comment by scopatz** [comment]
LGTM

File: .coveragerc [hunk 1]
```diff
@@ -4,4 +4,5 @@ omit =
   tests/*
   xonsh/__amalgam__.py
   */parser*_table.py
+  xonsh/ply/*
```
Comment (gforsyth) [inline .coveragerc]: Do we want coverage on ply?

**Comment by gforsyth** [comment]
Hey @laerus -- i'm good with this, just the one question about whether or not to include PLY in the coverage report

File: .coveragerc (hunk 1 above)
Comment (laerus) [inline .coveragerc]: nope that's why i've added it in the ommit section
am i missing something?

Comment (gforsyth) [inline .coveragerc]: My bad, early morning review is a bad idea

**Comment by scopatz** [comment]
also it is worth omitting lazyasd.py and amalgamate.py since these are tested in their own repos.

**Comment by laerus** [comment]
we also need to ommit all `__amalgam__.py` files in the subpackages

**Comment by laerus** [comment]
i don't think this needs a news entry

**Comment by gforsyth** [comment]
Thanks, @laerus!

**Pull request merged**